backend/clubs/models.py

```diff
@@ -698,6 +698,31 @@ def send_approval_email(self, request=None, change=False):
                 reply_to=settings.OSA_EMAILS + [settings.BRANDING_SITE_EMAIL],
             )
 
+    def get_latest_and_latest_approved_versions(self):
+        """
+        Returns the latest and latest approved versions of club key fields
+        [name, description, image] contents that trigger reapproval.
+        """
```

Comment:
Off topic: is it possible to limit how many historical records we store? Creating a new record for every change sounds expensive. I wonder if there's a way to limit it, e.g. only keep the last "approved" version of the club onwards.

Reply:
Was thinking about maybe storing the last k (maybe 3?) approved versions and pruning older ones on insert, since there isn't any need to store previous unapproved versions, but having a brief history of approvals could be marginally useful given that there are currently multiple admins who can approve clubs, and to account for accidental approvals etc.

Reply:
Sounds reasonable. We're already using the historical records in `ClubListSerializer.to_representation()`, to show the last approved version of a club to non-members. Agree that pruning older records would be great, but we're not introducing extra storage costs with this PR. I think it's safe to push it down the line.